### DISCUSSION TITLE
Can I make Intercom stop ringing after unlocking?

### DISCUSSION THREAD

**Discussion by stefanbd** [dgreif/ring#1242] [Q&A] [answered]
Currently, when I unlock the door using the API on Ring Intercom my Intercom continues ringing. However, if I answer the call through the app it stops. 

Is it possible to make the dinging cease once the door is unlocked using the API?

**Comment by dgreif** [comment, marked as answer]
We don't currently have the "answer call" piece of the API set up. I think that would be needed for the intercom to stop ringing. I don't have any plans to implement this piece myself, but @tsightler has mentioned possibly giving it a try
